src/System.Private.CoreLib/shared/System/Convert.Base64.Avx2.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+namespace System
+{
+    public static partial class Convert
+    {
+        // Based on "Base64 encoding with SIMD instructions" article by Wojciech Muła
+        // http://0x80.pl/notesen/2016-01-12-sse-base64-encoding.html
+
+
+        // SSSE3-based implementation
+
+        private static unsafe void EncodeBase64Sse(byte* input, int inputLength, byte* output, int outputLength)
```

Comment:
nit: Might be nice to name this one `Ssse3`, since that is the requirement for it